### ISSUE TITLE
Brave cannot be relaunched

### ISSUE THREAD

**Issue by Cepeel** [brave/browser-laptop#6465]
**Did you search for similar issues before submitting this one?**
Yes.

**Describe the issue you encountered:**
I install the latest release of the app, BraveSetup-x64.exe, and everything was running normally. After I closed the browser and tried to run it again, the app shows for a very short time (barely I can see the window) and then closes unexpectedly without an error. Any help on this?

**Expected behavior:**
App should be running.

- Platform (Win7, 8, 10? macOS? Linux distro?): 
Win7 x64

- Brave Version (revision SHA): 
0.12.15

- Any related issues: #1559

Thank you!

**Comment by jonathansampson** [comment]
@Cepeel Are you able to tell if Brave has any tabs opened (and what those tabs may be)?

**Comment by Cepeel** [comment]
I can't see, but right after the installation when the app worked, I closed all the tabs before closing the app.

**Comment by luixxiul** [comment]
Let me closing this in favor of #6504

**Comment by bbondy** [comment]
Does it launch if you start it directly in windows explorer from `%home%\AppData\Local\brave\app-0.12.15`?

**Comment by PhilNicholas** [comment]
I've updated to the 0.13.0 development release and the problem is gone. Whatever you did between the 0.12.15 and 0.13.0 versions seems to have resolved the issues. As a developer myself, I know that doesn't help a whole lot.  But I did everything I could think of to get 0.12.15 to run, but to no avail (i.e., it worked only once, after clean install, but never re-launched).
